lib/wings/active_fedora_converter/default_work.rb

```diff
@@ -137,6 +137,10 @@ def enforce_future_date_for_lease?
         false
       end
 
+      def file_sets
+        members.select(&:file_set?)
```

Comment:
Question: Is there no query scoped to file set members? Or do we just expect members to always be loaded here in entirety so it's more efficient to just iterate over them all?

Reply:
this is just a port of how `Hydra::Works::WorkBehavior` implements this: https://github.com/samvera/hydra-works/blob/c9b9dd0cf11de671920ba0a7161db68ccf9b7f6d/lib/hydra/works/models/concerns/work_behavior.rb#L90-L92